_site/authors/etagwerker/index.html

```diff
@@ -6,7 +6,7 @@
 
 
 
-    <title>Articles by Ernesto Tagwerker - The Lean Software Boutique</title>
+    <title>Articles by Etagwerker - The Lean Software Boutique</title>
```

Comment:
@etagwerker you should run `bundle update jekyll-authors` to get the latest version, and regenerate the site. I already did it in my other PR, #66, but just in case this gets merged before then.